platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -53,12 +51,28 @@
 
     private final String description;
 
+    private final String organization;
```

Comment:
Don't need these fields or any of the getters since they are all provided by `AbstractDescribable`.